website/addons/base/views.py

```diff
@@ -552,7 +552,7 @@ def addon_view_or_download_file(auth, path, provider, **kwargs):
 
             # Move file to trashed file node
             if not TrashedFileNode.load(file_node._id):
-                file_node.delete()
+                file_node.delete(user=auth.user)
```

Comment:
`user` should not be passed here.  This is the case where we have a StoredFileNode, but can't find in it in the metadata response.  i.e. it disappeared.